app/views/checkouts/_checkouts_table.html.erb

```diff
@@ -1,6 +1,29 @@
 <%= form_with url: renewals_batch_update_path, method: :patch, id: 'checkouts' do %>
     <table id="checkouts-table" class="table table-responsive-sm" role="table" aria-live="polite">
-      <caption><h2>Checkouts</h2></caption>
+      <caption>
+        <span class="dropdown float-right">
+          <style>
+            .dropdown button,
+            .dropdown-menu {
+              font-family: "Open Sans", Arial, serif;
+            }
+          </style>
```

Comment:
@ajkiessl I was able to fix the font by adding the styling here. I tried putting it in a few different stylesheets instead, but it didn't seem to work anywhere else.  I don't love it being in the html though so I'm open to suggestions

Reply:
You could maybe try adding a separate class to a stylesheet like `.dropdown-btn-font` and use the `!important` property with it.  So maybe something like this:

```
.dropdown-btn-font {
  font-family: "Open Sans", Arial, serif !important;
}
```

Then add that class to the dropdown menu or button.